instances/k8sworker/scripts/kubelet.service

```diff
@@ -21,6 +21,7 @@ ExecStart=/usr/bin/kubelet \
   --tls-private-key-file=/etc/kubernetes/ssl/apiserver-key.pem \
   --feature-gates=ExperimentalCriticalPodAnnotation=true \
   --eviction-hard=memory.available<500Mi,nodefs.available<2Gi,imagefs.available<2Gi \
+  --cloud-provider=external \
```

Comment:
Should also set `--provider-id` here so that we're not relying on the CCM to resolve the node name to OCID.